coveralls.json

```
@@ -1,6 +1,6 @@
 {
   "coverage_options": {
-    "minimum_coverage": 100.0
+    "minimum_coverage": 100
```

Can't remember why, but I started using floats instead of integers after seeing Coveralls failing unexpectedly.
```suggestion
    "minimum_coverage": 100.0
```